README.md

```diff
@@ -1,24 +1,69 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Getting started
 
-Things you may want to cover:
+You will need:
 
-* Ruby version
+* Ruby 2.7.x
+* Node 14.x
+* PostgreSQL 9.3 or above
+* [Yarn](https://yarnpkg.com/getting-started/install)
 
-* System dependencies
+We strongly recommend the use of [rbenv](https://github.com/rbenv/rbenv) and [nvm](https://github.com/nvm-sh/nvm) to manage ruby and node versions.
 
-* Configuration
+### Set up ruby
 
-* Database creation
+#### Using rbenv and bundler
 
-* Database initialization
+Run the following commands in the jingle-jam directory.
 
-* How to run the test suite
+    rbenv install
+    gem install bundler
+    bundle
 
-* Services (job queues, cache servers, search engines, etc.)
+#### Using rvm
 
-* Deployment instructions
+TODO
 
-* ...
+### Set up node
+
+    nvm install
+    npm install -g yarn
+    yarn install
+
+### Configure your database
+
+    createdb jingle_jam_development
```

Comment:
I would also mention that this requires postgres 9.5 running locally (or at least provide a docker file)